go.sum

```diff
@@ -215,6 +215,8 @@ github.com/cosiner/argv v0.1.0/go.mod h1:EusR6TucWKX+zFgtdUsKT2Cvg45K5rtpCcWz4hK
 github.com/cpuguy83/go-md2man v1.0.10/go.mod h1:SmD6nW6nTyfqj6ABTjUi3V3JVMnlJmwcJI5acqYI6dE=
 github.com/cpuguy83/go-md2man/v2 v2.0.0-20190314233015-f79a8a8ca69d/go.mod h1:maD7wRr/U5Z6m/iR4s+kqSMx2CaBsrgA7czyZG/E6dU=
 github.com/cpuguy83/go-md2man/v2 v2.0.1/go.mod h1:tgQtvFlXSQOSOSIRvRPT7W67SCa46tRHOmNcaadrF8o=
+github.com/crazycs520/genji/engine/badgerengine v0.12.1-0.20220328082424-727a2d089bde h1:URhoDN38NcI0UfTnCQZQ6h1254HPAlf/rDdwCDz03dI=
```

Comment:
looks like your fix is based on 0.12.1 instead of 0.13.0. Is it fine?

Reply:
looks weird, my fix is base on v0.13.0, the hash is right: `727a2d089bde`

![image](https://user-images.githubusercontent.com/26020263/160360120-75503afb-a01e-442c-9a29-e4a3e6f30220.png)